### PR TITLE
[MALI-1042] Save Image Activity Issue

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Saving an image results in the presented vc also to dismiss.
Replace UIActivityController with MiniAppActivityController which replaces the `dismiss` function

https://user-images.githubusercontent.com/17570451/172297520-747010de-12ba-457d-873c-c2c6010d6f96.MP4

## Links
https://jira.rakuten-it.com/jira/browse/MALI-1042

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

File: Example/Controllers/Extensions/ViewController+Activity.swift [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Foundation
+import UIKit
+
+public class MiniAppActivityController: UIActivityViewController {
+    public override func dismiss(animated flag: Bool, completion: (() -> Void)? = nil) {}
```
Comment (rleojoseph) [inline Example/Controllers/Extensions/ViewController+Activity.swift:5]: So we don't need to call `super` here 🤔

**Review by rleojoseph — COMMENTED** [review]
Also, could you please remove this https://github.com/rakutentech/ios-miniapp/blob/master/Sources/Classes/ui/MiniAppUI.swift#L115 extension?

Looks like you forgot to commit the Sample app project settings as well.

Also, could you please update Changelog about this fix, under sample app section

File: Example/Controllers/Extensions/ViewController+Activity.swift (hunk 1 above)
Comment (laubengaier) [reply Example/Controllers/Extensions/ViewController+Activity.swift:5]: that's the fix it doesn't dimiss anything but the activity vc (works like a joy)

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**